src/app/components/common/node-editor/insert-node/index.ts

```diff
@@ -94,8 +94,10 @@ async function removeOldConnection(node, editor) {
       inputConnId = element.id;
     }
   }
-  for (let t of target) {
-    await editor.addConnection(new Connection(connectionEvents, source, t));
+  if(source) {
+    for (let t of target) {
```

Comment:
It should be like-

`for (let target of targets) {`

Reply:
agree `targets` as plural!

`for (let target of targets) {`  or `for (let t of targets) {`